### DISCUSSION TITLE
Git.io shutdown

### DISCUSSION THREAD

**Discussion by dgw** [community/community#24621] [Other Feature Feedback, Questions, & Ideas]
<p>Can we get a bit more insight into <a href="https://github.blog/changelog/2022-01-11-git-io-no-longer-accepts-new-urls/" rel="noopener nofollow ugc">the shutdown of git.io</a>?</p>
<p>Some kind of timeline would be nice. Heck, it <em>would have been</em> nice to know in advance that new URLs would no longer be accepted at some future date. One does not simply expect a large company like Microsoft to turn stuff off without warning.</p>
<p>Above all, don’t forget that GitHub’s own Twitter account uses <code>git.io/c</code> in its bio. :stuck_out_tongue_winking_eye:</p>

**Comment by dgw** [comment]
<p>Bump, and also a proposal: submit the existing <code>git.io</code> shortlink mappings to the Internet Archive’s <a href="https://archive.org/details/301works" rel="noopener nofollow ugc">301Works project</a>.</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi there :wave: ! As of April 27, 2022, GitHub will now be preserving <a href="http://git.io">git.io</a> links in their read-only formats, these links will not be deprecated. Though we may remove individual links that point to spammy, malicious, or 404 links, we have decided to archive the current <a href="http://git.io">git.io</a> links in a new read-only service that will allow us to serve redirects for those links longer term. GitHub support will not be able to update or edit redirection records served by the <a href="http://git.io">git.io</a> archive service.</p>
<p>What began as an experiment in 2011, <a href="http://git.io/">git.io</a> was only lightly documented and thus not heavily used, however, <a href="http://git.io/">git.io</a> has increasingly been used for malicious purposes. Since January 2022, new link creation on <a href="http://git.io/">git.io</a> has ceased and we encourage users to make use of one of the many URL shortening services available with greater functionality than the <a href="http://git.io">git.io</a> service provided. To learn more, <a href="https://github.blog/changelog/2022-04-25-git-io-deprecation/">click here</a>.</p>

**Comment by dgw** [comment]
<p>Hi <a class="mention" href="/liligalante">@liligalante</a>,</p>
<p>Thanks for the update. I was dismayed by the blog post earlier this week that <a href="http://git.io" rel="noopener nofollow ugc">git.io</a> links would stop working in just four days. The update yesterday about transitioning to long-term read-only infrastructure instead of a hard shutdown was welcome news.</p>
<p>The stated reasons for no longer accepting new links don’t <em>really</em> make sense to me (how can a URL shortener be a security risk if it only allows links from GitHub-controlled domains?) but I do respect that I don’t have the full picture as a user—while your team has the entire millions-strong link database to analyze. :smiley_cat:</p>
<p>I’ll continue to help projects I see using these deprecated links move away from the service, but all of us appreciate GitHub keeping the links available anyway for use cases that can’t be easily changed!</p>

**Comment by queenofcorgis** [comment]
<p><a class="mention" href="/dgw">@dgw</a> thanks for sharing your perspective, it was members like you who helped us understand all the different use cases for these links and led to the 4/27 update.</p>
